SampleProject/TodayWidget/TodayViewController.m

```diff
@@ -0,0 +1,33 @@
+//
+//  TodayViewController.m
+//  TodayWidget
+//
+//  Created by Brock Boland on 11/3/16.
+//  Copyright © 2016 Vokal. All rights reserved.
+//
+
+#import "TodayViewController.h"
+#import <NotificationCenter/NotificationCenter.h>
+
+#import "VOKCoreDataManager.h"
+
+@interface TodayViewController () <NCWidgetProviding>
+
+@end
+
+@implementation TodayViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    //Make sure the widget compiles and can access Vokoder
+    [[VOKCoreDataManager sharedInstance] setResource:@"VOKCoreDataModel" database:@"VOKCoreDataModel.sqlite"];
```

Comment:
might be worth clarifiying that this is not accessing a shared container, but a core data database that is stored within the today extension.